### PR TITLE
added Might and Magic games, added Might and Magic remake, added KKnD…

### PR DESCRIPTION
**Pull request by gp-alex**
* added Might and Magic 6, 7, 8 to originals
 * added link to my remake of Might and Magic 6, 7, 8
 * added link to my remake of KKnD

### CODE REVIEW

**Comment by gp-alex** [comment]
hey sorry i dont have any images. the games look and play exactly as originals

**Comment by wonea** [comment]
@cxong Any ideas?

```
yaml.scanner.ScannerError: mapping values are not allowed here

  in "/home/travis/build/opengaming/osgameclones/originals/m.yaml", line 313, column 34
```

File: originals/m.yaml [hunk 1]
```diff
@@ -308,6 +308,57 @@
   names:
   - Midnight Club 2
 
+- name: 'Might and Magic VI: The Mandate of Heaven'
+  external:
+    wikipedia: Might and Magic VI: The Mandate of Heaven
```
Comment (cxong) [inline originals/m.yaml]:
```suggestion
    wikipedia: 'Might and Magic VI: The Mandate of Heaven'
```

File: originals/m.yaml [hunk 2]
```diff
@@ -308,6 +308,57 @@
...
+    - Fantasy
+    - Sci-Fi
+  names:
+  - 'Might and Magic 6: The Mandate of Heaven'
+  - Might and Magic 6 The Mandate of Heaven
+  - Might and Magic 6
+
+- name: 'Might and Magic VII: For Blood and Honor'
+  external:
+    wikipedia: Might and Magic VII: For Blood and Honor
```
Comment (cxong) [inline originals/m.yaml]:
```suggestion
    wikipedia: 'Might and Magic VII: For Blood and Honor'
```

File: originals/m.yaml [hunk 3]
```diff
@@ -308,6 +308,57 @@
...
+    - Fantasy
+    - Sci-Fi
+  names:
+  - 'Might and Magic 7: For Blood and Honor'
+  - Might and Magic 7 For Blood and Honor
+  - Might and Magic 7
+
+- name: 'Might and Magic VIII: Day of the Destroyer'
+  external:
+    wikipedia: Might and Magic VIII: Day of the Destroyer
```
Comment (cxong) [inline originals/m.yaml]:
```suggestion
    wikipedia: 'Might and Magic VIII: Day of the Destroyer'
```

**Pull request merged**